src/DssSpell.sol

```diff
@@ -20,25 +20,17 @@ pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dss/VestAbstract.sol";
 
 // import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
-interface GemLike {
-    function transfer(address, uint256) external returns (bool);
-    function approve(address, uint256) external returns (bool);
-    function allowance(address, address) external view returns (uint256);
+interface VestMKRLike {
```

Comment:
Can we use VestAbstract also instead of this?

Reply:
Yes, good catch.  Or I can yank the interfaces we need from the full abstract.  But we should use one or the other, not both.

Reply:
A subset abstract interface would be preferable I guess as only few are used.

Reply:
I changed this to `VestLike` and just reused `vest` between the DAI vest and MKR vest contracts in the spell.  You should review.